### PR TITLE
document the max_forward_iterations attribute

### LINKED ISSUE #195: max_forward_iterations not found in the online documentation
The `Diffractometer.max_forward_iterations` attribute does not appear in the online documentation except for the source code view.

Thanks @ambarb for pointing this out!

### PR DESCRIPTION
**Pull request by prjemian**
FIX #195

### CODE REVIEW

File: hkl/diffract.py [hunk 1]
```diff
@@ -98,6 +97,10 @@ class (using `calc_kw`) to instantiate a new one.
         (as used in `hkl.diffract.Diffractometer`), `calc_inst` must be
         specified upon initialization.
 
+    max_forward_iterations : integer (default 100)
```
Comment (mrakitin) [inline hkl/diffract.py:100]: Looks like the type should be `ophyd.Signal`, not an integer, as it's defined on the class level as (line 247 below):
```py
    max_forward_iterations = Cpt(Signal, value=100, kind="config")
```

**Review by mrakitin — COMMENTED** [review]
I think this docstring may need a correction.

File: hkl/diffract.py (hunk 1 above)
Comment (prjemian) [reply hkl/diffract.py:100]: Good catch.

**Comment by prjemian** [comment]
@mrakitin Ready for review.

**Review by mrakitin — APPROVED** [review]

**Pull request merged**